PickMeUp/PickMeUp/StringsList.h

```diff
@@ -0,0 +1,53 @@
+//
+//  StringsList.h
+//  PickMeUp
+//
+//  Created by Abigail Shilts on 7/5/22.
+//
+
+#ifndef StringsList_h
+#define StringsList_h
+
+static const NSString *const appID = @"appId";
```

Comment:
You shouldn't be keeping all constants in a single file. If they are local to specific place, just keep it there